sklearnex/svm/nusvc.py

```diff
@@ -83,6 +83,8 @@ def __init__(
     def fit(self, X, y, sample_weight=None):
         if sklearn_check_version("1.2"):
             self._validate_params()
+        elif self.nu <= 0 or self.nu > 1:
+            raise ValueError("nu <= 0 or nu > 1")
```

Comment:
Please add a comment that those checks are added to fix the failing test. I.e. need a description why only this parameter is checked here. And others like accuracy, number of iterations, etc. - are not checked.